dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -70,16 +70,19 @@ def __init__(self, options=None):
                 self.user_set_histogram_bin = bin_count_or_method
                 self.histogram_bin_method_names = ['custom']
         self.histogram_methods = {}
+        self._profile_histogram = None
         for method in self.histogram_bin_method_names:
             self.histogram_methods[method] = {
                 'total_loss': 0,
                 'current_loss': 0,
+                'suggested_bin_count': self.min_histogram_bin,
                 'histogram': {
                     'bin_counts': None,
                     'bin_edges': None
                 }
             }
-        self.quantiles = { bin_num: None for bin_num in range(1000) }
+        num_quantiles = 1000  # TODO: add to options
+        self.quantiles = {bin_num: None for bin_num in range(num_quantiles - 1)}
```

Comment:
Isn't this one less than before? Also does the math for this work out?

Reply:
Intentionally as 1000 quantiles would only have 999 bins. In the current code, the last bin is set to None and never gets updated.